### ISSUE TITLE
Non-unique IDs in DS section nav (WCAG Level A)

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2468]
DAC failed the DS website for its duplicate IDs (`id="section-navigation"`) on the vertical section nav. So we need to update the _page layout for the DS to use unique ID's for each section nav.

**Comment by jrbarnes9** [comment]
Will be resolved by #2466
